fuzzbench/test_e2e/end-to-end-test-config.yaml

```diff
@@ -0,0 +1,6 @@
+benchmarks:
+    - freetype2-2017
+    - libpng-1.2.56
```

Comment:
These two benchmarks are both non-oss-fuzz benchmarks.

Reply:
With the recent changes there should be no difference between standard and oss-fuzz benchmarks, both should work.

Reply:
Yes. I change it back to one normal and one oss-fuzz.